client/src/redux/alert/alertReducers.js

```diff
@@ -3,6 +3,7 @@ import { alertTypes } from './alertTypes';
 const initialState = {
   active: false,
   body: '',
+  displayAt: 'main', // either "main" or "modal", this determines the render location for the alert. This will reset whenever alertTypes.INACTIVE is dispatched
```

Comment:
@dzimmerman902 I came across a situation where I had to display alerts within a modal window, so I modified Alert to support that. Let me know if you have any feedback or questions on the implementation! I can walk you through this at the meeting on Monday too if that's easier.